WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -456,26 +457,81 @@ async Task<(AliceClient? AliceClient, PersonCircuit? PersonCircuit)> RegisterInp
 
 		roundState.LogDebug($"Inputs({smartCoins.Count()}) registration started - it will end in: {remainingTimeForRegistration:hh\\:mm\\:ss}.");
 
-		// Decrease the available time, so the clients hurry up.
+		// TODO: Having variations in these values between clients could harm privacy, should be round parameters
+		var hurryUpThreshold = 5;
```

Comment:
It makes sense if we are planning to change it dynamically from the backend. Do you think it will be required to adjust it remotely? Otherwise it is enough to have it like this as a constant local var.